src/python/foglamp/core/server_daemon.py

```diff
@@ -86,16 +86,24 @@ def start(cls):
         if pid:
             print("FogLAMP is already running in PID {}".format(pid))
         else:
-            # If it is desirable to output the pid to the console,
-            # os.getpid() reports the wrong pid so it's not easy.
-            print("Starting FogLAMP")
-
-            with daemon.DaemonContext(
-                working_directory=_WORKING_DIR,
-                umask=0o002,
-                pidfile=pidfile.TimeoutPIDLockFile(_PID_PATH)
-            ):
-                cls._start_server()
+            try:
```

Comment:
I don't think we need any of the daemon stuff related to pid files etc as it will be taken care of by the snap daemon.